static/app/components/replays/replayPlayer.tsx

```diff
@@ -87,15 +80,9 @@ function BasePlayerRoot({className, height = Infinity}: Props) {
 // Center the viewEl inside the windowEl.
 // This is useful when the window is inside a container that has large fixed
 // dimensions, like when in fullscreen mode.
-const SizingWindow = styled('div')<{isLoaded: boolean; minHeight: number}>`
+const SizingWindow = styled('div')`
   width: 100%;
   height: 100%;
-  ${p =>
-    p.isLoaded
-      ? ''
-      : p.minHeight !== Infinity
-      ? `min-height: ${p.minHeight}px !important;`
-      : ''}
```

Comment:
With this removed now the player will be `0px` tall when things are loading... use a placeholder instead of rendering this component in that case.